### PR TITLE
Makes it so that IRC can't be cloned

### PR DESCRIPTION
**Pull request by farie82**
## What Does This PR Do
Makes it so that IRC and such can't be cloned.
The made IRC had a holder brain that had a default species set. And the species was human....

## Why It's Good For The Game
Fixes a bug

## Changelog
:cl:
fix: IRCs can't be cloned
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -359,20 +359,20 @@
...
+		else
 			scantemp = "<span class=\"bad\">Error: Subject species is not scannable.</span>"
 			SSnanoui.update_uis(src)
 			return
 	if(subject.get_int_organ(/obj/item/organ/internal/brain))
 		var/obj/item/organ/internal/brain/Brn = subject.get_int_organ(/obj/item/organ/internal/brain)
 		if(istype(Brn))
 			if(NO_SCAN in Brn.dna.species.species_traits)
-				scantemp = "<span class=\"bad\">Error: [subject.dna.species.name_plural] are not scannable.</span>"
+				scantemp = "<span class=\"bad\">Error: [Brn.dna.species.name_plural] are not scannable.</span>"
```
Comment (farie82) [inline code/game/machinery/computer/cloning.dm]: It would show wrong information. Put a robo brain in a human and it would say "Humans are not scannable"

**Comment by Mitchs98** [comment]
This is beautiful.

**Comment by Ty-Omaha** [comment]
nice

**Comment by Spacemanspark** [comment]
The method we found this out with was hilarious.

"Oh great they're popping me into cloning, wonderful."

_Two minutes of grumbling later_

"Man I'm never going to be-"

_Spawns in again, cloned_

"WHAT"

File: code/game/machinery/computer/cloning.dm (hunk 1 above)
Comment (Crazylemon64) [reply code/game/machinery/computer/cloning.dm]: It sounds like the proper fix to that would be to set the species on the brain's DNA appropriately

Comment (Crazylemon64) [reply code/game/machinery/computer/cloning.dm]: though it looks like you've done that, too

**Comment by SynthNumerics** [comment]
i never even realised this was a thing, but fuck dude the code is beautiful

**Comment by Fox-McCloud** [comment]
Make sure this doesn't break MMI cloning.

You can robotize someone's chest, then yank out their brain, put it in an MMI, then put the MMI in the chest and revive them--they'll walk around and  be alive.

It's a bit convoluted, but people in such a state should still be able to be cloned and the DNA of the MMI should match that of the brain.

**Comment by farie82** [comment]
> Make sure this doesn't break MMI cloning.
> 
> You can robotize someone's chest, then yank out their brain, put it in an MMI, then put the MMI in the chest and revive them--they'll walk around and be alive.
> 
> It's a bit convoluted, but people in such a state should still be able to be cloned and the DNA of the MMI should match that of the brain.

I've not touched MMI code itself. Only the New of robo brains. I will still test it though just in case

**Comment by Fox-McCloud** [comment]
I just wanted to be sure, because MMI's as brain are really hacky and robotic brains are intertwined with them, so.

Thanks for confirming.

**Pull request merged**